### DISCUSSION TITLE
Compilation Error

### DISCUSSION THREAD

**Discussion by rarenu** [HASwitchPlate/openHASP#429] [General]
Hi, I am getting this error while compiling:
"src/hasp/hasp_font.cpp:14:10: fatal error: ft2build.h: No such file or directory"
Please advice.

**Comment by fvanroie** [comment]
Which version are you trying to build and for which environment?
Make sure you are on the latest `master` branch.

**Comment by rarenu** [comment]
Thank you for your response.
I am trying version 0.6.3 for d1-mini-esp32_ili931. I also tried for free
touchdeck_4MB environment but getting the same error.
Please check the screenshot attached to find where I am wrong and advise me.
Thank you for your help.
With kind regards.

**Comment by fvanroie** [comment]
Check out issue https://github.com/HASwitchPlate/openHASP/issues/405

`0.6.3` compilation is broken and won't be fixed due to time constraints.
Basically you need switch to the `master` branch, which should compile fine.
